src/encoded/commands/create_mapping_on_deploy.py

```diff
@@ -123,6 +123,85 @@
     'Page',
 ]
 
+WEBPROD = 'fourfront-webprod'
+WEBPROD_TWO = 'fourfront-webprod2'
```

Comment:
Some reason WEBPROD2 is a bad name? If you grep (case-insensitive) for webprod2, it'd be nice to find it in the ordinary way.

Reply:
I used it, I think it will work